_templates/page.html

```diff
@@ -47,7 +47,7 @@
   {# Open Graph metadata #}
   <meta property="og:image" content="https://docs.citusdata.com/en/stable/_images/citus-docs-og-logo.png" />
   <meta property="og:title" content="{{ title }} - {{ docstitle }}" />
-  <meta property="og:description" content="Docs for the Citus extension to Postgres. Citus distributes your data &amp; queries across multiple nodes in a cluster, so your database can scale and your queries are fast." />
+  <meta property="og:description" content="Docs for the Citus database, an open source extension to Postgres. Citus gives you the Postgres you love at any scale—from a single node to a large distributed cluster. Available as 100% open source &amp; as a managed service with Azure Cosmos DB for PostgreSQL." />
```

Comment:
For HTML descriptions I reco they be of 250-310 chars or so in length, but front loaded to the first 150 chars so that the most important bits show up in Google SERP. (Last I checked, Google sometimes chooses its own meta description anyway so fine tuning this too much can be a waste of time, as Google will serve up what it deems best for the description in SERP.) 

I will take a peek we should reorder the contents of the HTML description so it's more front loaded, but whether keeping the a longer description (to benefit slack users, etc) is worth it. Stay tuned.

Reply:
I updated the HTML description, it's still longer than you've requested but that's because I believe the extra description will serve us well in non-Google SERP scenarios. See what you think!